### ISSUE TITLE
Please, do not overwrite the status bar text

### ISSUE THREAD

**Issue by schittli** [halirutan/IntelliJ-Key-Promoter-X#97]
**Thank you very much for sharing your great work!** ❤️ 

**Describe the bug**
I classify it as a bug because the idea of the key promoter is probably to gently teach the user keyboard shortcuts while not blocking the user in his work.

However, because the IDE displays very important information in the status bar text and the Key Promoter overwrites this information, most developers are likely to be hindered in their work by this otherwise ingenious plugin.

**To Reproduce**
Steps to reproduce the behaviour:
1. Install the key promoter 😄 
2. Wait until it suggests to use a keyboard shortcut
3. The key promoter overwrites the IntelliJ Status Bar text and if there was important information, then you'll have to search for it somewhere else 😞 

**Expected behaviour**
* Please, do not overwrite the status bar text or just display it for e.g. 3 seconds.
* The key promoter also informs about the popup text, which I find a very clever solution because you can also deactivate messages. That's Great!

**Screenshots**

![image](https://user-images.githubusercontent.com/8282673/187047935-a9ae7933-f5c5-4f15-878e-5a605c4d3b66.png)

**Please also provide:**
 - OS: Windows 10 x64
 - IntelliJ Product:
 - IntelliJ IDEA 2021.2 (Community Edition)
Build #IC-212.4746.92, built on July 27, 2021

Thanks a lot, kind regards,
Thomas

**Comment by halirutan** [comment]
Yep, I agree with your assessment, and I have seen this in the past, but not anymore. I never set the status bar explicitly, and it seems to be IntelliJ Platform's choice to display some notifications in the status bar.

Could you do me a favor and try IntelliJ IDEA 2022.2.1? I can't reproduce the issue with this version and maybe the SDK already changed the behavior for information notifications.

**Comment by halirutan** [comment]
Nope, I saw this issue now even in the most recent version. I need to determine if I can influence that notifications are shown in the status bar.

**Comment by schittli** [comment]
Thank you very much, @halirutan, to investigate this issue.

**Comment by halirutan** [comment]
I believe I tracked this down finally. However, currently there seems to be no way from stopping notifications to show up in the status bar.

https://github.com/JetBrains/intellij-community/blob/1a2ef409244bfc4984eb455821631fc869ebfdf0/platform/platform-impl/src/com/intellij/notification/LogModel.java#L61-L80

I'll ask a colleague
